src/cloud/release-notes/mcc-release-notes.md

```diff
@@ -15,18 +15,20 @@ The `{{site.data.var.mcc}}` package uses the following version sequence: `<major
 
 This release includes the following updates:
 
--  Item 1
+- {:.new}Extended the functionality of the WARM_UP_PAGES environment variable to support cache preloading for specific product pages. See the [post-deploy variables]({{site.baseurl}}/cloud/env/variables-post-deploy.html#warm_up_pages) topic for a detailed feature description.<!--MAGECLOUD-4444-->
```

Comment:
In this topic there are several ways the variable name is emphasized, sometimes as inline code, sometimes bold. Does this one need some emphasis?